adyen.php

```diff
@@ -619,6 +614,51 @@ public function hookPaymentOptions()
             }
         }
 
+        if (!empty($paymentMethods['paymentMethods'])) {
+            foreach ($paymentMethods['paymentMethods'] as $paymentMethod) {
+                $issuerList = [];
+                if (empty($paymentMethod['type']) || $paymentMethod['type'] == 'scheme') {
+                    continue;
+                }
+                if (isset($paymentMethod['details'])) {
```

Comment:
```suggestion
                if (!empty($paymentMethod['details'])) {
```

So if it is an empty array or not set then we won't even try to foreach through it.

Reply:
In this case, I don't assume the performance impact is going to matter. I assume the compiler will just skip this if it's an empty array.

We can enforce some similar conditions by adding a check for `is_array`. I just prefer to make the condition obvious instead of abstract.

Reply:
I would prefer !empty() over isset in this case as it just do an extra check if it has a value. is_array is indeed a bit overkill